src/index.html

```diff
@@ -40,16 +39,31 @@
       padding-top: 50px;
       padding-bottom: 50px;
     }
+    .col-md-4 {
+      float: center;
+      margin-left: auto;
+      margin-right: auto;
+      position: relative;
+      left: 75px;
```

Comment:
You're setting left and centering. Seems odd. Please check whether the "left: 75px" is needed (or doing anything).